modules/@apostrophecms/schema/ui/apos/scss/AposInputArray.scss

```diff
@@ -253,9 +253,7 @@
 }
 
 .apos-input-array-inline-body {
-  overflow: hidden;
   max-height: 999px;
-  padding: $spacing-base 0;
```

Comment:
Remove the padding here to add it above to `.apos-schema` it allows the animation of  the dropdown to not jump (padding still being here when finishing to close the dropdown).
It's more pleasing to the eye.